infrastructure/main.tf

```diff
@@ -48,6 +53,62 @@ module "wa_task_management_api_database" {
   sku_name           = var.database_sku_name
 }
 
+//New Azure Flexible database
+module "wa_task_management_api_database_flexible" {
+  providers = {
+    azurerm.postgres_network = azurerm.postgres_network
+  }
+
+  source             = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=postgres-vnet-provider"
+  product            = var.product
+  component          = var.component
+  name               = "${var.postgres_db_component_name}-postgres-db-flexible"
+  location           = var.location
+  business_area      = var.business_area
+  env                = var.env
+  pgsql_databases = [
+    {
+      name : var.postgresql_database_name
+    }
+  ]
+  pgsql_server_configuration = [
+    {
+      name  = "wal_level"
+      value = "logical"
+    }
+  ]
+
+  pgsql_version      = 14
+  common_tags        = local.common_tags
+
+  admin_user_object_id = var.jenkins_AAD_objectId
+
+}
+
+//New Azure Flexible database replica
+module "wa_task_management_api_database_flexible_replica" {
+  providers = {
+    azurerm.postgres_network = azurerm.postgres_network
+  }
+
+  source             = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=postgres-vnet-provider"
+  product            = var.product
+  component          = var.component
+  name               = "${var.postgres_db_component_name}-postgres-db-flexible-replica"
+  location           = var.location
+  business_area      = var.business_area
+  env                = var.env
+  pgsql_databases = [
+    {
+      name : var.postgresql_database_name
+    }
+  ]
+  pgsql_version      = 14
+  common_tags        = local.common_tags
+
+  admin_user_object_id = var.jenkins_AAD_objectId
+
+}
```

Comment:
This all looks good.
May be a question for PlatOps but will all the above config stay the same for when we want to go to prod or were there some temporary references to get it working in perftest?